lib/generators/litestack/install/install_generator.rb

```diff
@@ -0,0 +1,26 @@
+class Litestack::InstallGenerator < Rails::Generators::Base
+  source_root File.expand_path("templates", __dir__)
+
+  def modify_database_adapater
+    gsub_file "config/database.yml", "adapter: sqlite3", "adapter: litedb"
+  end
+
+  def modify_cache_store_adapter
+    gsub_file "config/environments/production.rb",
+      "# config.cache_store = :mem_cache_store",
+      "config.cache_store = :litecache, { path: './path/to/your/cache/file' }"
```

Comment:
May I suggest for being more opinionated about the path of the cache? I think something like `Rails.root.join("tmp/cache/litestack-#{Rails.env}.db")` would _probably_ be enough for most use cases.

I understand this is the current suggestion on `README` but perhaps that could change too.

WDYT?

Reply:
Yeah, as I said in the other comment above I want to be much more opinionated about Litestack databases living in a `/data/#{Rails.env}/queue.db`, etc. directory. Putting all the databases in one directory and making the paths predictable will make backups and restores a more sane proposition.

I'll implement a configuration object for Litestack that handles the database paths for the various framework integrations, and each path will also support procs to deal with sharding, multi-region deployments, etc.